src/main/webapp/app/pages/curation/collapsible/CancerTypeCollapsible.tsx

```diff
@@ -204,19 +211,21 @@ function CancerTypeCollapsible({
           action={
             <>
               <CommentIcon id={`${cancerTypesUuid}_diagnostic_comments`} path={`${cancerTypePath}/diagnostic_comments`} />
-              <RCTButton cancerTypePath={cancerTypePath} relevantCancerTypesInfoPath={`${cancerTypePath}/diagnostic`} />
+              <RCTButton cancerTypePath={cancerTypePath} relevantCancerTypesInfoPath={`${cancerTypePath}/diagnostic`} readOnly={readOnly} />
```

Comment:
Passing in the readOnly props explicitly is not required because it will be injected by mobx.